Jamfile

```diff
@@ -9,13 +9,14 @@
 project boost-gil
     :
     requirements
-        <toolset>intel:<debug-symbols>off
```

Comment:
Did you mean to remove this instruction ? Seems unrelated...

Reply:
No, not at all.

I shuffled the lines moving `<toolset>msvc:` to the top, so the `# MSVC` comments is in the beginning

Reply:
Oh, I see. I had missed that before. Thanks.

Reply:
No problem. Thx for extra pair of eyes